mlir/lib/Dialect/Affine/IR/AffineOps.cpp

```diff
@@ -3044,7 +3116,7 @@ static void composeSetAndOperands(IntegerSet &set,
 LogicalResult AffineIfOp::fold(FoldAdaptor, SmallVectorImpl<OpFoldResult> &) {
   auto set = getIntegerSet();
   SmallVector<Value, 4> operands(getOperands());
-  composeSetAndOperands(set, operands);
+  composeSetAndOperands(set, operands, /*composeAffineMin=*/false);
```

Comment:
Isn't it by default false?

Reply:
Yes, I used compiler errors to guide me to all the places I needed to fix to properly plug the bool in, I can clean up now, thanks for catching!